targets/tizenrt-artik053/README.md

```diff
@@ -94,30 +81,36 @@ jerryscript/tools/build.py \
     --toolchain=${PWD}/jerryscript/cmake/toolchain_mcu_artik053.cmake
 ```
 
-**Note**: there is a Makefile in the `targets/tizenrt-artik053/` folder that also helps to build JerryScript for TizenRT.
+Alternatively, there is a Makefile in the `targets/tizenrt-artik053/` folder that also helps to build JerryScript for TizenRT.
 
 ```
 # assuming you are in jerry-tizenrt folder
 $ cd jerryscript
 $ make -f targets/tizenrt-artik053/Makefile.tizenrt
 ```
 
-#### 6. Build TizenRT binary
+#### 5. Add your JavaScript program to TizenRT (optional)
+
+If you have script files for JerryScript, you can add them to TizenRT.
+These files will be flashed into the target's `/rom` folder.
+Note that your content cannot exceed 1200 KB.
 
 ```
 # assuming you are in jerry-tizenrt folder
-$ cd TizenRT/os
-$ make
+cp jerryscript/tests/hello.js TizenRT/tools/fs/contents/
```

Comment:
Could you please change this line to `$ echo "Hello JerryScript!" > TizenRT/tools/fs/contents/hello.js`? I forgot to update this line :)

Reply:
Actually, you did change that line but I've changed it further. That `echo` command would not write a useful JS program into hello.js (as it'd only contain a string literal without anything printing it). So, I've decided to simply copy a working hello.js from the existing tests instead.